portal/verify_session.php

```diff
@@ -18,12 +18,12 @@
 
 // All of the common intialization steps for the get_* patient portal functions are now in this single include.
 
-
-
 //continue session
 // Will start the (patient) portal OpenEMR session/cookie.
-require_once(__DIR__ . "/../src/Common/Session/SessionUtil.php");
-OpenEMR\Common\Session\SessionUtil::portalSessionStart();
+if (session_status() === PHP_SESSION_NONE) {
```

Comment:
this also makes me nervous :)
What bug are we fixing on this? (I'll take a quick look)

Reply:
when login to portal top of `signer_modal.php` starts a portal session but then requires verify_session.php which then notifies that a session has already been started 

![Screenshot from 2022-06-14 11-02-37](https://user-images.githubusercontent.com/5618372/173616641-dd870402-f998-4f24-9a7a-6e9240c7650d.png)

it's just a notice so can ignore and move on if easiest

Reply:
Just get rid of verify_session in signer_modal because the session will always be started if the conditional is true.